.github/workflows/publish-clients.yml

```diff
@@ -14,14 +14,22 @@ jobs:
 
     - name: Generate client
       run: |
-        docker run --rm -v ${GITHUB_WORKSPACE}:/local openapitools/openapi-generator-cli generate -i /local/api/notification-service.yaml -g javascript -o /local/client -p projectName="@commitdev/zero-notification-service-client" -p projectDescription="Generated client for zero-notification-service" -p projectVersion="${{ github.event.release.tag_name }}" -p moduleName="NotificationService"
+        docker run --rm -v ${GITHUB_WORKSPACE}:/local openapitools/openapi-generator:cli-5.3.x \
```

Comment:
this also had the newer generated schema with the `inner` in response types so have to cap the version